core/contracts/VoteTiming.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * @title VoteTiming
+ * @dev Computes rounds and phases for an equal length commit-reveal voting cycle. 
+ */
+library VoteTiming {
+    using SafeMath for uint;
+
+    // Note: the phases must be in order. Meaning the first enum value must be the first phase, etc.
+    enum Phase {
+        Commit,
+        Reveal
+    }
+
+    // Note: this MUST match the number of values in the enum above.
+    uint private constant NUM_PHASES = 2;
+
+    struct Data {
+        uint roundId;
+        uint roundStartTime;
+        uint phaseLength;
+    }
+
+    function init(Data storage data, uint phaseLength) internal {
+        data.phaseLength = phaseLength;
+        data.roundId = 1;
+        data.roundStartTime = 0;
+    }
+
+    function getLastUpdatedRoundId(Data storage data) internal view returns (uint) {
+        return data.roundId;
+    }
+
+    function shouldUpdateRoundId(Data storage data, uint currentTime) internal view returns (bool) {
```

Comment:
So this function is in plain English:  "has the stored round ended"
and `updateRoundId` is: "start new round"

Reminds me a bunch of my struggle with understanding "previous", "current", and "next" in the TokenizedDerivative :)

Reply:
Do you have a suggestion for a different set of names? I don't really like the ones I went with, so suggestions are welcome :)